app/controllers/api_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ApiController < ActionController::API
+
+    private
+
+      def parsed_user
+        user = request.env['HTTP_AUTHORIZATION'].match(/^Bearer (.*)/)[1]
+        JSON.parse(jwt_decode(user)[:user])
+      end
+
+      def current_user
+        @current_user ||= User.find(parsed_user["id"])
+      end
+
+      def jwt_decode(token)
+        decoded = JWT.decode(token, Rails.application.secrets.secret_key_base)
```

Comment:
What's the behavior here if the token is invalid or expired? Does it raise? if it does should we handle it?

Reply:
Yes, I don't have error checks yet. I'll add those. Right now we just get a Rails error.

Reply:
Error rescuing in place. Simple for now, but will catch all JWT decode errors.